### PR TITLE
Adds 4.8.13 RNs for 9-27

### PR DESCRIPTION
**Pull request by stevsmit**
No BZ. 

RNs 4.8.13 for 9-27. 

Preview: https://deploy-preview-36771--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-13

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -2674,3 +2674,29 @@ The minimum storage required to install an {product-title} cluster has decreased
...
+
+[id="ocp-4-8-13-features"]
+==== Features
+
+* Kubernetes v1.21.4 is now available. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1213[v1.21.4], link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1212[v1.21.3], and link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1211[v1.21.2]. 
+
+[id="ocp-4-8-13-bug-fixes"]
+==== Bug fixes
+
+* Previously, there was an unchecked index operation on a slice when using the `--max components` argument. Consequently, the `oc` client returned a `panic: runtime error: slice bounds out of range` error and crashed. This update adds a check to ensure that a value is not requested for an index that is out of range. As a result, when using the `--max-components` argument, the `oc` client will no longer crash.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=2004193[*BZ#2004193*])
```
Comment (ctauchen) [inline release_notes/ocp-4-8-release-notes.adoc:2697]:
```suggestion
* Previously, there was an unchecked index operation on a slice when using the `--max components` argument. Consequently, the `oc` client returned a panic error and crashed. This update adds a check to ensure that a value is not requested for an index that is out of range. As a result, when using the `--max-components` argument, the `oc` client no longer crashes.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=2004193[*BZ#2004193*])
```

**Comment by ctauchen** [comment]
@stevsmit All good, with a few suggestions.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -2674,3 +2674,30 @@ The minimum storage required to install an {product-title} cluster has decreased
...
+{product-title} release 4.8.13, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3632[RHBA-2021:3632] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:3631[RHSA-2021:3631] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6363171[{product-title} 4.8.13 container image list]
+
+[id="ocp-4-8-13-features"]
+==== Features
+
+* Kubernetes v1.21.4 is now available. More information can be found in the following changelogs: link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1213[v1.21.4], link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1212[v1.21.3], and link:https://github.com/kubernetes/kubernetes/blob/release-1.21/CHANGELOG/CHANGELOG-1.21.md#changelog-since-v1211[v1.21.2].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-8-release-notes.adoc:2692]: Not critical, but we usually don't put a "v" in front of the versions. I'd just say "Kubernetes 1.21.4..."

**Review by bergerhoffer — COMMENTED** [review]
Just one more comment, not critical.

**Comment by bergerhoffer** [comment]
LGTM, merging!

**Pull request merged**